crates/hyperdrive-math/src/utils.rs

```diff
@@ -98,6 +98,31 @@ pub fn calculate_initial_bond_reserves(
         .mul_down(inner)
 }
 
+/// @dev Calculates the number of base that are not reserved by
+///      open positions.
+/// @param vault_share_price The current vault share price.
+/// @param share_reserves The current share reserves, from market state
+/// @param long_exposure The current long exposure, from market state
+/// @param minimum_share_reserves The minimum share reserves, from pool config
+/// @return idleShares The amount of shares that are available for LPs to
+///         withdraw.
```

Comment:
We don't typically include NatSpec in Rust. When you add this to `lp.rs`, can you follow the comment convention of that file? You can probably just take out the contents of the dev and make that the whole comment.